ui/packages/consul-ui/app/components/topology-metrics/index.hbs

```diff
@@ -25,8 +25,9 @@
       @hasMetricsProvider={{@hasMetricsProvider}}
       @noMetricsReason={{this.noMetricsReason}}
     >
-    {{#if (and @hasMetricsProvider (not-eq @service.Service.Kind 'ingress-gateway'))}}
+    {{#if (and @hasMetricsProvider this.mainNotIngressService (not-eq item.Kind 'ingress-gateway'))}}
       <TopologyMetrics::Stats
+        data-test-downstream-metrics-stats
         @nspace={{or item.Namespace 'default'}}
```

Comment:
Could you add a comment here somewhere explaining that this is one of the only places in the UI where is ok to default to the `default` namespace? I'm pretty sure that's a correct statement?

Reply:
I've added the comment.